apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -865,15 +879,15 @@ func (ctx *prepareOrGetContext) SetError(idx int, err *params.Error) {
 }
 
 func (ctx *prepareOrGetContext) ProcessOneContainer(
-	env environs.Environ, callContext context.ProviderCallContext, idx int, host, container *state.Machine,
+	env environs.Environ, callContext context.ProviderCallContext, idx int, host, guest Machine,
 ) error {
-	containerId, err := container.InstanceId()
+	containerId, err := guest.InstanceId()
 	if ctx.maintain {
 		if err == nil {
 			// Since we want to configure and create NICs on the
 			// container before it starts, it must also be not
 			// provisioned yet.
-			return errors.Errorf("container %q already provisioned as %q", container, containerId)
+			return errors.Errorf("container %q already provisioned as %q", guest, containerId)
```

Comment:
Is it appropriate to use an type Machine interface {} as a %q in a format string? I thought that only worked as a way to quote raw strings.
We probably need to check the error formatting here and probably have something other than %q, or maybe guest.SOMETHING()

Reply:
Done. New test for it was a breeze with the new suite too :)